ToDo/src/components/Tasks/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {
+  FlatList,
+  KeyboardAvoidingView,
+  Platform,
+  StyleSheet,
+  View,
+} from 'react-native';
+import {useAppSelector} from '../../hooks/hooks';
+import Task from '../Task';
+
+function Tasks() {
+  const tasks = useAppSelector(state => state.tasks);
```

Comment:
We should always move our selector function out of components to keep references stable. See https://react-redux.js.org/api/hooks#equality-comparisons-and-updates